src/main/java/org/alliancegenome/curation_api/interfaces/crud/AlleleCrudInterface.java

```diff
@@ -46,4 +46,16 @@ public void reindex(
 		@DefaultValue("1") @QueryParam("typesToIndexInParallel") Integer typesToIndexInParallel
 	);
 
+	
+	@Override
+	@PUT
+	@Path("/")
+	@JsonView(View.FieldsAndLists.class)
+	public ObjectResponse<Allele> update(Allele entity);
+	
+	@Override
+	@POST
+	@Path("/")
+	@JsonView(View.FieldsAndLists.class)
```

Comment:
Are these going to cause issues with circular references?

Reply:
@oblodgett Shouldn't do at this point, but have added allele views for future-proofing